.vagrant/machines/default/virtualbox/action_provision

```diff
@@ -0,0 +1 @@
+1.5:aaa5b7f4-2181-4b5f-9f45-f0429ceafe22
```

Comment:
Make a new commit in your for removing that file. No need for a new PR. same goes for pyc files